frontend/apps/filemanager/filemanager.lua

```diff
@@ -243,8 +244,18 @@ function FileManager:init()
                     end,
                 }
             },
+            {},
```

Comment:
Visual effect of grouping for functionalities? If that's expected, better document in the code.